src/model/SubSurface.cpp

```diff
@@ -147,6 +147,11 @@ namespace detail {
       this->resetAdjacentSubSurface();
     }
 
+    // Remove it from the extensible groups in ShadingControl(s)
+    for (auto& sc: shadingControls()) {
+      sc.removeSubSurface(this->getObject<SubSurface>());
+    }
+
```

Comment:
`SubSurface::remove` will remove itself from ShadingControl(s) so we don't end up with empty fields in the extensible portion. @joseph-robertson FYI